### PR TITLE
[IN-219][Reviews] Ember-Component-CSS -> Ember-CSS-Modules

### PR DESCRIPTION
**Pull request by chrisseto**
Switch from Ember-Component-CSS to Ember-CSS-Modules as Ember-Component-CSS does not support Ember-Engines.

QA Notes:
Almost every style sheet was touched in this PR, but there should be no noticeable changes.
The one exception being that the status headers on the submissions page will no longer become bold if you right click on them. (See the below screenshot)

screen shot 2018-06-07 at 3 41 10 pm

### CODE REVIEW

File: app/components/notification-list-row/styles.scss
Comment (laurenbarker) [inline app/components/notification-list-row/styles.scss]: Delete?

File: ember-cli-build.js
Comment (laurenbarker) [inline ember-cli-build.js]: Delete?

File: ember-cli-build.js
Comment (laurenbarker) [inline ember-cli-build.js]: Also, delete?

**Review by laurenbarker — DISMISSED** [review]
Could you add a PR description with QA notes? 

Other than the random commented out stuff and the lack of a description this looks great. Thanks for also adding style linting! 

Were there styles you ended up actually changing?

**Comment by chrisseto** [comment]
I cleaned up some of the styles but everything should look the same.

**Comment by chrisseto** [comment]
side by side: http://recordit.co/OD7omLPFr6
Looking into the moderators tab and preprint preview page oddities.

**Comment by laurenbarker** [comment]
The skeleton screens for the provider section on the dashboard got stretched.
Before:
providers-normal

After:
providers-stretched

**Comment by chrisseto** [comment]
Nice catch!

File: app/components/action-feed-entry/styles.scss
Comment (laurenbarker) [inline app/components/action-feed-entry/styles.scss]: Why does the class need to be in the stylesheet? Do unused styles get removed on compile or something?

Comment (chrisseto) [reply app/components/action-feed-entry/styles.scss]: css modules munges all the class names when the app is built. It's in the style sheet so the tests can get the "compiled" name. I could just leave it untouched, as a `class` rather than `local-class` but it just looks a bit odd. Or do a weird selector. Whichever works for me.

Comment (laurenbarker) [reply app/components/action-feed-entry/styles.scss]: I don't really have a preference. I'll leave that up to @alexschiller

**Comment by alexschiller** [comment]
Looked everything over. Fixed merge conflict. Checked out PR locally and everything looked good. Will wait for TCI to be green after the conflict fix and then get it merged.

**Review by alexschiller — APPROVED** [review]
wew

**Comment by chrisseto** [comment]
Thanks @alexschiller

**Pull request merged**
